components/layout/construct.rs

```diff
@@ -222,7 +248,7 @@ impl InlineFragmentsAccumulator {
                 pseudo: node.get_pseudo_element_type().strip(),
                 style: node.style(style_context),
                 selected_style: node.selected_style(),
-                flags: InlineFragmentNodeFlags::empty(),
+                flags: FIRST_FRAGMENT_OF_ELEMENT | LAST_FRAGMENT_OF_ELEMENT,
```

Comment:
I concur with @Permutatrix; let's make things as simple as possible. But *definitely* comment this explaining how things work and the rationale.